Xamarin.PropertyEditing.Mac/Controls/BindingEditor/BindingEditorWindow.cs

```diff
@@ -0,0 +1,274 @@
+						}
+					}
+				}
+
+				if (e.PropertyName == nameof (CreateBindingViewModel.ShowResourceSelector)) {
+					if (this.viewModel.ShowResourceSelector) {
+						typeHeader.Title = Properties.Resources.SelectResourceTitle;
+					}
+				}
+			};
+
+			this.viewModel.CreateValueConverterRequested += OnCreateValueConverterRequested;
+
+			this.ButtonDone.Activated += (sender, e) => {
+				if (pathSelectorControl.CustomPath.Enabled && !string.IsNullOrEmpty (pathSelectorControl.CustomPath.Cell.Title)) {
+					this.viewModel.Path = pathSelectorControl.CustomPath.Cell.Title;
+				}
+
+				Close ();
+			};
+
+			// More Settings
+			var controlTop = 6;
+			foreach (PropertyViewModel item in this.viewModel.BindingProperties) {
+
+				var label = new UnfocusableTextField {
+					StringValue = item.Property.Name,
+					TranslatesAutoresizingMaskIntoConstraints = false,
+				};
+				this.MoreSettingsView.AddSubview (label);
+
+				this.MoreSettingsView.AddConstraints (new[] {
+					NSLayoutConstraint.Create (label, NSLayoutAttribute.Top, NSLayoutRelation.Equal, this.MoreSettingsView, NSLayoutAttribute.Top, 1f, controlTop),
+					NSLayoutConstraint.Create (label, NSLayoutAttribute.Left, NSLayoutRelation.Equal, this.MoreSettingsView, NSLayoutAttribute.Left, 1f, 16f),
+					NSLayoutConstraint.Create (label, NSLayoutAttribute.Height, NSLayoutRelation.Equal, 1f, 18f),
+				});
+
+				var textField = new NSTextField {
+					Enabled = item.Property.CanWrite,
+					StringValue = item.Name,
+					TranslatesAutoresizingMaskIntoConstraints = false,
+				};
+				textField.Changed += (sender, e) => {
+					// TODO What do we update in the VM with this value?
+				};
+
+				this.MoreSettingsView.AddSubview (textField);
+
+				controlTop += PropertyEditorControl.DefaultControlHeight;
+
+				this.MoreSettingsView.AddConstraints (new[] {
+					NSLayoutConstraint.Create (textField, NSLayoutAttribute.Top, NSLayoutRelation.Equal, label, NSLayoutAttribute.Bottom, 1f, 5f),
+					NSLayoutConstraint.Create (textField, NSLayoutAttribute.Left, NSLayoutRelation.Equal, this.MoreSettingsView, NSLayoutAttribute.Left, 1f, 16f),
+					NSLayoutConstraint.Create (textField, NSLayoutAttribute.Width, NSLayoutRelation.Equal, 1f, 100f),
+				});
+
+				controlTop += PropertyEditorControl.DefaultControlHeight;
+			}
+
+			var boundsHeight = controlTop;
+
+			controlTop = 9;
+			foreach (PropertyViewModel item in this.viewModel.FlagsProperties.Cast<PropertyViewModel> ()) {
+
+				var flag = new NSButton {
+					ControlSize = NSControlSize.Small,
+					Enabled = item.Property.CanWrite,
+					Font = NSFont.FromFontName (PropertyEditorControl.DefaultFontName, PropertyEditorControl.DefaultFontSize),
+					Title = item.Property.Name,
+					TranslatesAutoresizingMaskIntoConstraints = false,
+				};
+
+				flag.SetButtonType (NSButtonType.Switch);
+				flag.Activated += (sender, e) => {
+					// TODO What do we update in the VM with this value?
```

Comment:
@ermau Same here. I'm finding it difficult to pin point exactly which property on the VM this should hook up to. Any pointers?